src/acuvity/sdk.py

```diff
@@ -71,7 +71,7 @@ def __init__(
                 server_url = utils.template_url(server_url, url_params)
         server_defaults: List[Dict[str, str]] = [
             {
-                "apex_domain": apex_domain or "api.apex.acuvity.ai",
+                "apex_domain": apex_domain or "apex.acuvity.ai",
```

Comment:
change this back please. My PR which had that change for the `acuvity` repo didn't land because Antoine didn't like other things in there.

Reply:
got it. I think speakeasy changed this.